src/main/java/net/dv8tion/jda/client/entities/Group.java

```diff
@@ -39,9 +40,9 @@
     List<User> getUsers();
     List<User> getNonFriendUsers();
     List<Friend> getFriends();
-
-    //getGroupManager()
-
+    
+    GroupManager getGroupManager();
```

Comment:
sorry a lot of classes didn't have a getManagerUpdatable() method
at least the ones I looked at for example didn't